src/util/adb.js

```diff
@@ -7,7 +7,8 @@ import {
   WebExtError,
 } from '../errors';
 import {createLogger} from '../util/logger';
-import packageIdentifiers from '../firefox/package-identifiers';
+import packageIdentifiers, { defaultApkComponents }
+  from '../firefox/package-identifiers';
```

Comment:
Nitpick: would you mind to reformat this as:
```
import packageIdentifiers, {
  defaultApkComponents,
} from '../firefox/package-identifiers';
```

Just to keep it consistent with the convention we used for some other multiline imports already part of the sources.